add-ons/istio/README.adoc

```diff
@@ -0,0 +1,84 @@
+:linkattrs:
+
+= Istio Add-on
+
+This add-on deploys https://Istio.io/[Istio] service mesh on to Minishift-provisioned OpenShift cluster.
+
+== Minishift profile setup
+
+The standard Istio deployment creates few pods and its related resources. 
+
+Considering this, it is ideal to setup a minishift profile with the following configuration:
+
+[code,sh]
+----
+$ minishift profile set servicemesh <1>
+$ minishift config set memory 8GB <2>
+$ minishift config set cpus 3 <3>
+$ minishift config set openshift-version v3.7.0 <4>
+$ minishift addon enable admin-user <5>
+$ minishift addon enable anyuid <6>
+$ minishift start <7>
+----
+
+<1> Create a new Minishift profile named **servicemesh**
+<2> Use 8GB of memory for istio and apps that you might deploy later
+<3> Since Istio minimum deploys approximately __9__ pods, 3 cpus will give a max of 30 pods
```

Comment:
Nice addition of context. :thumbsup:

I'd still rewrite this just a little bit, though:

```adoc
<3> 3 CPUs results in a maximum of 30 pods since Istio deploys approximately 9 pods at minimum
```